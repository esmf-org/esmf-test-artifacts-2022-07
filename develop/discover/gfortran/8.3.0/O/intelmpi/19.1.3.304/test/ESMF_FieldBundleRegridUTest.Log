build time -- 2022-06-21 01:07:22.019535
20220621 011945.098 INFO             PET0 !!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!
20220621 011945.098 INFO             PET0 !!! THE ESMF_LOG IS SET TO OUTPUT ALL LOG MESSAGES !!!
20220621 011945.098 INFO             PET0 !!!     THIS MAY CAUSE SLOWDOWN IN PERFORMANCE     !!!
20220621 011945.098 INFO             PET0 !!! FOR PRODUCTION RUNS, USE:                      !!!
20220621 011945.098 INFO             PET0 !!!                   ESMF_LOGKIND_Multi_On_Error  !!!
20220621 011945.098 INFO             PET0 !!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!
20220621 011945.098 INFO             PET0 Running with ESMF Version   : v8.4.0b01-2-gf0d4ab211d
20220621 011945.098 INFO             PET0 ESMF library build date/time: "Jun 21 2022" "01:06:02"
20220621 011945.098 INFO             PET0 ESMF library build location : /gpfsm/dnb04/projects/p98/mpotts/esmf/gfortran_8.3.0_intelmpi_O_develop
20220621 011945.098 INFO             PET0 ESMF_COMM                   : intelmpi
20220621 011945.099 INFO             PET0 ESMF_MOAB                   : enabled
20220621 011945.099 INFO             PET0 ESMF_LAPACK                 : enabled
20220621 011945.099 INFO             PET0 ESMF_NETCDF                 : disabled
20220621 011945.099 INFO             PET0 ESMF_PNETCDF                : disabled
20220621 011945.099 INFO             PET0 ESMF_PIO                    : disabled
20220621 011945.099 INFO             PET0 ESMF_YAMLCPP                : enabled
20220621 011945.100 INFO             PET0  Beginning Test, file ESMF_FieldBundleRegridUTest.F90, line 56
20220621 011945.100 INFO             PET0  NUMBER_OF_PROCESSORS           4
20220621 011945.190 INFO             PET0  PASS  FieldBundleRegrid between a 0 to 360 sphere and a -180 to 180 sphere, ESMF_FieldBundleRegridUTest.F90, line 67
20220621 011945.277 INFO             PET0  PASS  FieldBundleRegrid between a 0 to 360 sphere and a -180 to 180 sphere, ESMF_FieldBundleRegridUTest.F90, line 74
20220621 011945.285 INFO             PET0  PASS  FieldBundleRegrid sphere with mask, ESMF_FieldBundleRegridUTest.F90, line 90
20220621 011945.377 INFO             PET0  PASS  FieldBundleRegrid between a 0 to 360 sphere and a LocStream, ESMF_FieldBundleRegridUTest.F90, line 97
20220621 011945.377 INFO             PET0  Ending Test, file ESMF_FieldBundleRegridUTest.F90, line 102
20220621 011945.377 INFO             PET0 Finalizing ESMF
20220621 011945.099 INFO             PET1 !!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!
20220621 011945.099 INFO             PET1 !!! THE ESMF_LOG IS SET TO OUTPUT ALL LOG MESSAGES !!!
20220621 011945.099 INFO             PET1 !!!     THIS MAY CAUSE SLOWDOWN IN PERFORMANCE     !!!
20220621 011945.099 INFO             PET1 !!! FOR PRODUCTION RUNS, USE:                      !!!
20220621 011945.099 INFO             PET1 !!!                   ESMF_LOGKIND_Multi_On_Error  !!!
20220621 011945.099 INFO             PET1 !!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!
20220621 011945.099 INFO             PET1 Running with ESMF Version   : v8.4.0b01-2-gf0d4ab211d
20220621 011945.099 INFO             PET1 ESMF library build date/time: "Jun 21 2022" "01:06:02"
20220621 011945.099 INFO             PET1 ESMF library build location : /gpfsm/dnb04/projects/p98/mpotts/esmf/gfortran_8.3.0_intelmpi_O_develop
20220621 011945.099 INFO             PET1 ESMF_COMM                   : intelmpi
20220621 011945.099 INFO             PET1 ESMF_MOAB                   : enabled
20220621 011945.099 INFO             PET1 ESMF_LAPACK                 : enabled
20220621 011945.099 INFO             PET1 ESMF_NETCDF                 : disabled
20220621 011945.099 INFO             PET1 ESMF_PNETCDF                : disabled
20220621 011945.099 INFO             PET1 ESMF_PIO                    : disabled
20220621 011945.099 INFO             PET1 ESMF_YAMLCPP                : enabled
20220621 011945.100 INFO             PET1  Beginning Test, file ESMF_FieldBundleRegridUTest.F90, line 56
20220621 011945.100 INFO             PET1  NUMBER_OF_PROCESSORS           4
20220621 011945.190 INFO             PET1  PASS  FieldBundleRegrid between a 0 to 360 sphere and a -180 to 180 sphere, ESMF_FieldBundleRegridUTest.F90, line 67
20220621 011945.277 INFO             PET1  PASS  FieldBundleRegrid between a 0 to 360 sphere and a -180 to 180 sphere, ESMF_FieldBundleRegridUTest.F90, line 74
20220621 011945.285 INFO             PET1  PASS  FieldBundleRegrid sphere with mask, ESMF_FieldBundleRegridUTest.F90, line 90
20220621 011945.377 INFO             PET1  PASS  FieldBundleRegrid between a 0 to 360 sphere and a LocStream, ESMF_FieldBundleRegridUTest.F90, line 97
20220621 011945.377 INFO             PET1  Ending Test, file ESMF_FieldBundleRegridUTest.F90, line 102
20220621 011945.377 INFO             PET1 Finalizing ESMF
20220621 011945.100 INFO             PET2 !!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!
20220621 011945.100 INFO             PET2 !!! THE ESMF_LOG IS SET TO OUTPUT ALL LOG MESSAGES !!!
20220621 011945.100 INFO             PET2 !!!     THIS MAY CAUSE SLOWDOWN IN PERFORMANCE     !!!
20220621 011945.100 INFO             PET2 !!! FOR PRODUCTION RUNS, USE:                      !!!
20220621 011945.100 INFO             PET2 !!!                   ESMF_LOGKIND_Multi_On_Error  !!!
20220621 011945.100 INFO             PET2 !!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!
20220621 011945.100 INFO             PET2 Running with ESMF Version   : v8.4.0b01-2-gf0d4ab211d
20220621 011945.100 INFO             PET2 ESMF library build date/time: "Jun 21 2022" "01:06:02"
20220621 011945.100 INFO             PET2 ESMF library build location : /gpfsm/dnb04/projects/p98/mpotts/esmf/gfortran_8.3.0_intelmpi_O_develop
20220621 011945.100 INFO             PET2 ESMF_COMM                   : intelmpi
20220621 011945.100 INFO             PET2 ESMF_MOAB                   : enabled
20220621 011945.100 INFO             PET2 ESMF_LAPACK                 : enabled
20220621 011945.100 INFO             PET2 ESMF_NETCDF                 : disabled
20220621 011945.100 INFO             PET2 ESMF_PNETCDF                : disabled
20220621 011945.100 INFO             PET2 ESMF_PIO                    : disabled
20220621 011945.100 INFO             PET2 ESMF_YAMLCPP                : enabled
20220621 011945.100 INFO             PET2  Beginning Test, file ESMF_FieldBundleRegridUTest.F90, line 56
20220621 011945.100 INFO             PET2  NUMBER_OF_PROCESSORS           4
20220621 011945.190 INFO             PET2  PASS  FieldBundleRegrid between a 0 to 360 sphere and a -180 to 180 sphere, ESMF_FieldBundleRegridUTest.F90, line 67
20220621 011945.277 INFO             PET2  PASS  FieldBundleRegrid between a 0 to 360 sphere and a -180 to 180 sphere, ESMF_FieldBundleRegridUTest.F90, line 74
20220621 011945.285 INFO             PET2  PASS  FieldBundleRegrid sphere with mask, ESMF_FieldBundleRegridUTest.F90, line 90
20220621 011945.377 INFO             PET2  PASS  FieldBundleRegrid between a 0 to 360 sphere and a LocStream, ESMF_FieldBundleRegridUTest.F90, line 97
20220621 011945.377 INFO             PET2  Ending Test, file ESMF_FieldBundleRegridUTest.F90, line 102
20220621 011945.377 INFO             PET2 Finalizing ESMF
20220621 011945.099 INFO             PET3 !!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!
20220621 011945.099 INFO             PET3 !!! THE ESMF_LOG IS SET TO OUTPUT ALL LOG MESSAGES !!!
20220621 011945.099 INFO             PET3 !!!     THIS MAY CAUSE SLOWDOWN IN PERFORMANCE     !!!
20220621 011945.099 INFO             PET3 !!! FOR PRODUCTION RUNS, USE:                      !!!
20220621 011945.099 INFO             PET3 !!!                   ESMF_LOGKIND_Multi_On_Error  !!!
20220621 011945.099 INFO             PET3 !!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!
20220621 011945.099 INFO             PET3 Running with ESMF Version   : v8.4.0b01-2-gf0d4ab211d
20220621 011945.099 INFO             PET3 ESMF library build date/time: "Jun 21 2022" "01:06:02"
20220621 011945.099 INFO             PET3 ESMF library build location : /gpfsm/dnb04/projects/p98/mpotts/esmf/gfortran_8.3.0_intelmpi_O_develop
20220621 011945.099 INFO             PET3 ESMF_COMM                   : intelmpi
20220621 011945.099 INFO             PET3 ESMF_MOAB                   : enabled
20220621 011945.099 INFO             PET3 ESMF_LAPACK                 : enabled
20220621 011945.099 INFO             PET3 ESMF_NETCDF                 : disabled
20220621 011945.099 INFO             PET3 ESMF_PNETCDF                : disabled
20220621 011945.099 INFO             PET3 ESMF_PIO                    : disabled
20220621 011945.099 INFO             PET3 ESMF_YAMLCPP                : enabled
20220621 011945.100 INFO             PET3  Beginning Test, file ESMF_FieldBundleRegridUTest.F90, line 56
20220621 011945.100 INFO             PET3  NUMBER_OF_PROCESSORS           4
20220621 011945.190 INFO             PET3  PASS  FieldBundleRegrid between a 0 to 360 sphere and a -180 to 180 sphere, ESMF_FieldBundleRegridUTest.F90, line 67
20220621 011945.277 INFO             PET3  PASS  FieldBundleRegrid between a 0 to 360 sphere and a -180 to 180 sphere, ESMF_FieldBundleRegridUTest.F90, line 74
20220621 011945.285 INFO             PET3  PASS  FieldBundleRegrid sphere with mask, ESMF_FieldBundleRegridUTest.F90, line 90
20220621 011945.377 INFO             PET3  PASS  FieldBundleRegrid between a 0 to 360 sphere and a LocStream, ESMF_FieldBundleRegridUTest.F90, line 97
20220621 011945.377 INFO             PET3  Ending Test, file ESMF_FieldBundleRegridUTest.F90, line 102
20220621 011945.377 INFO             PET3 Finalizing ESMF
