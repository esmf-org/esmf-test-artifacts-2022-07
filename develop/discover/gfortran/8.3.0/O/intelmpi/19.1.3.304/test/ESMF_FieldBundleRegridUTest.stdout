build time -- 2022-07-16 01:17:55.246803
srun: cluster configuration lacks support for cpu binding
  Beginning Test, file ESMF_FieldBundleRegridUTest.F90, line 56
  Beginning Test, file ESMF_FieldBundleRegridUTest.F90, line 56
  NUMBER_OF_PROCESSORS           4
  Beginning Test, file ESMF_FieldBundleRegridUTest.F90, line 56
  Beginning Test, file ESMF_FieldBundleRegridUTest.F90, line 56
  NUMBER_OF_PROCESSORS           4
  NUMBER_OF_PROCESSORS           4
  NUMBER_OF_PROCESSORS           4
  PASS  FieldBundleRegrid between a 0 to 360 sphere and a -180 to 180 sphere, ESMF_FieldBundleRegridUTest.F90, line 67
  PASS  FieldBundleRegrid between a 0 to 360 sphere and a -180 to 180 sphere, ESMF_FieldBundleRegridUTest.F90, line 67
  PASS  FieldBundleRegrid between a 0 to 360 sphere and a -180 to 180 sphere, ESMF_FieldBundleRegridUTest.F90, line 67
  PASS  FieldBundleRegrid between a 0 to 360 sphere and a -180 to 180 sphere, ESMF_FieldBundleRegridUTest.F90, line 67
 utest src field name = src360_L                                                        
 utest dst field name = dst360_B                                                        
 utest src field name = src360_L                                                        
 utest dst field name = dst360_B                                                        
 utest src field name = src360_L                                                        
 utest src field name = src360_L                                                        
 utest dst field name = dst360_B                                                        
 utest dst field name = dst360_B                                                        
 utest src field name = src360_G                                                        
 utest dst field name = dst360_Y                                                        
 utest src field name = src360_G                                                        
 utest src field name = src360_G                                                        
 utest dst field name = dst360_Y                                                        
 utest dst field name = dst360_Y                                                        
 utest src field name = src360_G                                                        
 utest src field name = src360_X                                                        
 utest dst field name = dst360_T                                                        
 utest src field name = src360_X                                                        
 utest dst field name = dst360_T                                                        
 utest dst field name = dst360_Y                                                        
 utest src field name = src360_X                                                        
 utest src field name = src360_A                                                        
 utest dst field name = dst360_S                                                        
 utest src field name = src360_A                                                        
 utest dst field name = dst360_S                                                        
 utest dst field name = dst360_T                                                        
 utest src field name = src360_X                                                        
 utest dst field name = dst360_T                                                        
 utest src field name = src360_C                                                        
 utest dst field name = dst360_P                                                        
 utest src field name = src360_C                                                        
 utest dst field name = dst360_P                                                        
 utest src field name = src360_A                                                        
 utest dst field name = dst360_S                                                        
 utest src field name = src360_A                                                        
 utest dst field name = dst360_S                                                        
 utest src field name = src360_H                                                        
 utest dst field name = dst360_N                                                        
  PASS  FieldBundleRegrid between a 0 to 360 sphere and a -180 to 180 sphere, ESMF_FieldBundleRegridUTest.F90, line 74
 utest src field name = src360_C                                                        
 utest src field name = src360_C                                                        
 utest dst field name = dst360_P                                                        
 utest dst field name = dst360_P                                                        
 utest src field name = src360_H                                                        
 utest src field name = src360_H                                                        
 utest dst field name = dst360_N                                                        
 utest dst field name = dst360_N                                                        
  PASS  FieldBundleRegrid between a 0 to 360 sphere and a -180 to 180 sphere, ESMF_FieldBundleRegridUTest.F90, line 74
 utest src field name = src360_H                                                        
 utest dst field name = dst360_N                                                        
  PASS  FieldBundleRegrid between a 0 to 360 sphere and a -180 to 180 sphere, ESMF_FieldBundleRegridUTest.F90, line 74
  PASS  FieldBundleRegrid between a 0 to 360 sphere and a -180 to 180 sphere, ESMF_FieldBundleRegridUTest.F90, line 74
  PASS  FieldBundleRegrid sphere with mask, ESMF_FieldBundleRegridUTest.F90, line 90
  PASS  FieldBundleRegrid sphere with mask, ESMF_FieldBundleRegridUTest.F90, line 90
  PASS  FieldBundleRegrid sphere with mask, ESMF_FieldBundleRegridUTest.F90, line 90
  PASS  FieldBundleRegrid sphere with mask, ESMF_FieldBundleRegridUTest.F90, line 90
  PASS  FieldBundleRegrid between a 0 to 360 sphere and a LocStream, ESMF_FieldBundleRegridUTest.F90, line 97
  PASS  FieldBundleRegrid between a 0 to 360 sphere and a LocStream, ESMF_FieldBundleRegridUTest.F90, line 97
  Ending Test, file ESMF_FieldBundleRegridUTest.F90, line 102
  PASS  FieldBundleRegrid between a 0 to 360 sphere and a LocStream, ESMF_FieldBundleRegridUTest.F90, line 97
  Ending Test, file ESMF_FieldBundleRegridUTest.F90, line 102
  PASS  FieldBundleRegrid between a 0 to 360 sphere and a LocStream, ESMF_FieldBundleRegridUTest.F90, line 97
  Ending Test, file ESMF_FieldBundleRegridUTest.F90, line 102
  Ending Test, file ESMF_FieldBundleRegridUTest.F90, line 102
  PET           0  Test Elapsed Time    271.89600000000002       msec.
  PET           1  Test Elapsed Time    270.67400000000004       msec.
  PET           2  Test Elapsed Time    271.27999999999997       msec.
  PET           3  Test Elapsed Time    267.31300000000005       msec.
